reframe/core/logging.py

```diff
@@ -222,7 +222,7 @@ def _create_syslog_handler(site_config, config_prefix):
     except ValueError:
         pass
     else:
-        address = (host, port)
+        address = (host, int(port))
```

Comment:
We should handle the case when `port` cannot be converted to an `int`.

Reply:
That's true.